java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3676,4 +3677,82 @@ public void getbit(BaseClient client) {
         executionException = assertThrows(ExecutionException.class, () -> client.getbit(key2, 1).get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void bitop(BaseClient client) {
```

Comment:
Can we add the following tests?
- passing an empty key list, an exception should be thrown (Sho has asked me to test this before)
- passing multiple keys with the NOT operator (exception should be thrown)
- I think we need to add a test that an error is thrown when cross-slot keys are passed, but that should be added to the test in cluster/CommandTests.java instead of this file

Reply:
Added thanks!